pkg/bloombuild/planner/planner.go

```diff
@@ -476,7 +473,7 @@ func (p *Planner) loadTenantWork(
 
 		// If this is the first this we see this table, initialize the map
 		if tenantTableWork[table] == nil {
-			tenantTableWork[table] = make(map[string][]v1.FingerprintBounds, tenants.Len())
+			tenantTableWork[table] = make(map[string][]v1.FingerprintBounds, tenants.Remaining())
```

Comment:
I don't know this code too well, but I assume the behavior of `Len` and `Remaining` here is the same – i.e. `Remaining` doesn't do additional filtering, for example.

Reply:
Yup :)